lib/src/configs/forms/wysiwyg.vue

```diff
@@ -15,11 +17,65 @@ export default {
   components: {
     quillEditor,
   },
+  props: {
+    locales: {
+      type: Array,
+      default() {
+        return ['en', 'fr']
+      }
+    },
+    editorOptions: {
+      type: Object,
+      default() {
+        return {
+          modules: {
+            toolbar: [
+              ['bold', 'italic', 'underline', 'strike'],        // toggled buttons
```

Comment:
why are these toolbar lists duplicated on multiple vue components ?

Reply:
I removed from the main index.vue file as it is fed by a prop from `_.vue` and `Sections.vue` components.
And the reason why I am using it on multiple components is caused by the dynamic page feature where I had to set its default prop on the `_.vue`. And keep it configurable by the host project when using a static section page while having a default value on `Sections.vue` component. Same for the wysiwyg component: giving it a default value and making it configurable through a prop when called from the host project